### ISSUE TITLE
Fix Text Spacing in Astronomy Metadata Fields

### ISSUE THREAD

**Issue by posixeleni** [IQSS/dataverse#2622]
Field Display Name: RedshiftType -> Redshift Type (also fix in the field description as well)

Controlled Vocabulary for Type:
- EventList -> Event List
- LightCurve -> Light Curve
- PrettyPicture -> Pretty Picture
- ValuePair -> Value Pair

Also a good opportunity to add identifiers in the controlled vocabulary

**Comment by pdurbin** [comment]
Still a bug as of 4.6.1. Here's the file: https://github.com/IQSS/dataverse/blob/v4.6.1/scripts/api/data/metadatablocks/astrophysics.tsv

#2551 is related.

**Comment by johnhuck** [comment]
Enquiring minds are curious about the source of the controlled vocabulary for astrotype.

**Comment by pdurbin** [comment]
@johnhuck here you go:

https://github.com/IQSS/dataverse/blob/v4.18/scripts/api/data/metadatablocks/astrophysics.tsv#L30

Pull requests welcome! 😄

**Comment by johnhuck** [comment]
thanks, @pdurbin - yes, but where did this list of type values originate? is it from a standard? was it created locally?

**Comment by pdurbin** [comment]
@johnhuck oh! For that you'd go to http://guides.dataverse.org/en/4.18/user/appendix.html#metadata-references

<img width="1266" alt="Screen Shot 2019-11-14 at 8 13 31 PM" src="https://user-images.githubusercontent.com/21006/68909105-4452a600-071b-11ea-8572-cde738e0ab1f.png">

**Comment by johnhuck** [comment]
@pdurbin Of course! I'm sorry, I thought we'd already checked there. Looks like the terms are primarily from the VO schema, rather than the IVOA (there's some overlap between the VO and IVOA types, but not much, which is curious).

I don't see the values `Figure`, `PrettyPicture`, `Documentation`, and `Press Release` in the VO list (although IVOA has `Press`), but the VO document says the list is extensible, so I guess we can assume they were introduced somewhere along the way. Perhaps not important at this point. Thanks for the assist!

**Comment by cmbz** [comment]
To focus on the most important features and bugs,     we are closing issues created before 2020 (version 5.0)     that are not new feature requests with the label 'Type: Feature'.    

If you created this issue and you feel the team     should revisit this decision, please reopen the issue and leave a comment.

**Comment by jggautier** [comment]
I was asked to propose next steps based on discussion in the related pull request that @sbondka created at https://github.com/IQSS/dataverse/pull/10206.

I think that we should work with experts, like contacts at Harvard's CfA and folks who worked on that VO metadata standard, to figure out how the vocab terms should appear when researchers choose them, and to figure out how identifiers, a blockUri and/or termUris should be added to the metadata block.

@scolapasta suggested closing the PR, and that we could reopen it if we decide to build on it. I agreed.

We'll meet with @cmbz next week to review this.

**Comment by sbarbosadataverse** [comment]
> I was asked to propose next steps based on discussion in the related pull request that [@sbondka](https://github.com/sbondka) created at [#10206](https://github.com/IQSS/dataverse/pull/10206).
> 
> I think that we should work with experts, like contacts at Harvard's CfA and folks who worked on that VO metadata standard, to figure out how the vocab terms should appear when researchers choose them, and to figure out how identifiers, a blockUri and/or termUris should be added to the metadata block.
> 
> [@scolapasta](https://github.com/scolapasta) suggested closing the PR, and that we could reopen it if we decide to build on it. I agreed.
> 
> We'll meet with [@cmbz](https://github.com/cmbz) next week to review this.

@jggautier @kmika11 is interested in working with CfA-maybe she can pursue this?

**Comment by sbarbosadataverse** [comment]
@kmika11 Can you review this issue and let me know if this is something you are comfortable pursuing? Thanks

**Comment by kmika11** [comment]
Happy to help! From this issue I can't tell if the current problem is with the display of the existing fields or whether there is interest in updating the astronomy metadata block with additional or revised fields.

**Comment by sbarbosadataverse** [comment]
> Happy to help! From this issue I can't tell if the current problem is with the display of the existing fields or whether there is interest in updating the astronomy metadata block with additional or revised fields.

@scolapasta can you please identify the exact issue that needs fixing here? Katie is willing to investigate with CfA! Thanks

**Comment by scolapasta** [comment]
@kmika11 I think your question is actually a good summary of what I understand the issue to be - a PR had been opened to fix the display of existing fields and @jggautier was suggesting that it would be better to do a more comprehensive review (with the proper experts) of the block as a whole. @jggautier can correct me if that understanding is wrong.

**Comment by jggautier** [comment]
Heya, that sounds like it, at least at a high level. I'm not able to revisit this though

**Comment by cmbz** [comment]
2025/02/24
- Next steps
  - @kmika11 please take a look at the comments in: https://github.com/IQSS/dataverse/pull/10206
  - @sbarbosadataverse and @kmika11 please review together and decide next steps

**Comment by cmbz** [comment]
@sbarbosadataverse will review.
